ghcup-vanilla-0.0.8.yaml

```diff
@@ -4329,14 +4329,113 @@ ghcupDownloads:
         dlUri: https://downloads.haskell.org/~ghc/9.8.2/ghc-9.8.2-testsuite.tar.xz
+    9.10.1:
+      viArch:
+        A_32:
+          Linux_Debian:
+            unknown_versioning: &ghc9101-i386-deb10
+              dlHash: 5367e692dd9efd9062846334ec6aeeddf186c01b1551b3207649a2ce16f4df76
+              dlSubdir: ghc-9.10.1-i386-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-i386-deb10-linux.tar.xz
+          Linux_Mint:
+            unknown_versioning: *ghc9101-i386-deb10
+          Linux_Ubuntu:
+            unknown_versioning: *ghc9101-i386-deb10
+          Linux_UnknownLinux:
+            unknown_versioning: *ghc9101-i386-deb10
+        A_64:
+          Darwin:
+            unknown_versioning:
+              dlHash: 8cf22188930e10d7ac5270d425e21a3dab606af73a655493639345200c650be9
+              dlSubdir: ghc-9.10.1-x86_64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-x86_64-apple-darwin.tar.xz
+          Linux_Alpine:
+            unknown_versioning:
+              dlHash: 327b424d30524a9d2dfe569e1ef72b8967675c555ce684ef60deec1233e3f4f5
+              dlSubdir: ghc-9.10.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-x86_64-alpine3_12-linux.tar.xz
+          Linux_CentOS:
+            ( >= 7 && < 8 ): &ghc9101-x86_64-centos7
+              dlHash: fab143f10f845629cb1b373309b5680667cbaab298cf49827e383ee8a9ddf683
+              dlSubdir: ghc-9.10.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-x86_64-centos7-linux.tar.xz
+            unknown_versioning: *ghc9101-x86_64-centos7
+          Linux_Debian:
+            (>= 10 && < 11):
+              dlHash: 7fbdca8971c89a54cff8e88aee825c2ef2bcc9f7a050ee1e1e844763e39c361f
+              dlSubdir: ghc-9.10.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-x86_64-deb10-linux.tar.xz
+            < 10: &ghc9101-x86_64-deb9
+              dlHash: 5c4fb1c2185320afd6b1efaa155f97e956fa782004ce829fb41879ad0fd1cd14
+              dlSubdir: ghc-9.10.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-x86_64-deb9-linux.tar.xz
+            '>= 11': &ghc9101-x86_64-deb11
+              dlHash: 78975575b8125ecf1f50f78b1016b14ea6e87abbf1fc39797af469d029c5d737
+              dlSubdir: ghc-9.10.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-x86_64-deb11-linux.tar.xz
+            unknown_versioning: *ghc9101-x86_64-deb11
+          Linux_Fedora:
+            '>= 33':
+              dlHash: eef58297f48cae3d8003fd6e8bc295b6e42fe982ad902eada99a72d559384cab
+              dlSubdir: ghc-9.10.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-x86_64-fedora33-linux.tar.xz
+            unknown_versioning: *ghc9101-x86_64-centos7
+          Linux_Mint:
```

Comment:
we should probably also have an `unknown_versioning: *ghc-9101-x86_64-ubuntu20_04` here just in case `/etc/os-release` contains garbage or something.